content/integrations/sumologic.md

```diff
@@ -13,11 +13,12 @@ Enable the Sumo Logic integration to:
 
 ## Installation
 
-There are two parts to the SumoLogic integration, based on the direction of information between the products. You can send alerts and events from Datadog to Sumologic and you can have SumoLogic send data to Datadog.
+Datadog integration with SumoLogic is two-part: send alerts and events from Datadog to Sumologic, or have SumoLogic send data to Datadog. You can enable one or both.
```

Comment:
How about:

"Datadog integrates with Sumo Logic in two ways: you can forward Sumo Logic log data to your Datadog event stream, and/or you can use Sumo Logic as a notification channel from Datadog alerts and events. In other words, either service can inform the other."